invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -422,6 +422,29 @@ dl.details-list {
   text-align: center;
 }
 
+.text-align-right {
+  text-align: right;
+
+  &-mobile {
+    @media all and (max-width: @largestMobileScreen) {
+      text-align: right;
```

Comment:
why is it needed for each device? if you make
```
.text-align-right{
  text-align: right !important;
}
```
(as we use important in the utility classes, like this one, it should do the job for all devices

Reply:
It's needed if the alignment is different on different devices.